From: "Barak A. Pearlmutter" <barak+git@pearlmutter.net>
Date: Wed, 22 Feb 2017 08:46:47 +0000
Subject: autoconf AS_HELP_STRING

---
 configure.ac | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/configure.ac b/configure.ac
index 6dfd778..7713fb1 100644
--- a/configure.ac
+++ b/configure.ac
@@ -43,7 +43,7 @@ AH_TEMPLATE(ENABLE_EDITING, [Enable the EditEnhMetaFile() function])
 # disable threads
 
 AC_ARG_ENABLE(debug,
-[  --enable-debug          enable libEMF debugging [default=no]],
+AS_HELP_STRING([--enable-debug],[enable libEMF debugging [default=no]]),
   enable_debug=$enableval, enable_debug=no)
 if test "x$enable_debug" = xyes; then
   AC_DEFINE(LIBEMF_DEBUG, 1)
@@ -57,7 +57,7 @@ your CFLAGS])
 fi
 
 AC_ARG_ENABLE(threads,
-[  --enable-threads           enable threads [default=yes]],
+AS_HELP_STRING([--enable-threads],[enable threads [default=yes]]),
   have_threads=$enableval, have_threads=yes)
 if test "x$have_threads" = xyes; then
 THREAD_LIBS=
@@ -83,7 +83,7 @@ fi
 # Adds a lot of code and data to the library, though.
 
 AC_ARG_ENABLE(editing,
-[  --enable-editing     enable EditEnhMetaFile() function [default=no]],
+AS_HELP_STRING([--enable-editing],[enable EditEnhMetaFile() function [default=no]]),
   enable_editing=$enableval, enable_editing=no)
 if test "x$enable_editing" = xyes; then
    AC_DEFINE(ENABLE_EDITING, 1)
